docs/man/docker-logs.1.md

```diff
@@ -35,6 +36,9 @@ then continue streaming new output from the container’s stdout and stderr.
 **--tail**="all"
    Output the specified number of lines at the end of logs (defaults to all logs)
 
+**--truncate**=*true*|*false*
+    Fetch current logs and truncate to zero of a container
```

Comment:
Can you rearrange the wording here and below to something more like - 

`Fetch the current container logs and then truncate to zero length`

I do wonder if `truncate to zero length` is too much of a technical implementation detail - though it communicates the concurrency effects correctly.